firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-# Unreleased (21.4.0)
+# Unreleased 
+- [changed] Internal changes to the in-memory representation of Firestore
```

Comment:
This should be a past-tense sentence. Something like:

> Changed the in-memory representation of Firestore documents to reduce memory allocation and improve performance. Users of `DocumentSnapshot.getData()` and `DocumentSnapshot.toObject()` will see the biggest improvement.

WDYT?

Reply:
Changed to:

```
Changed the in-memory representation of Firestore documents to
  reduce memory allocations and improve performance. Calls to 
  `DocumentSnapshot.getData()` and `DocumentSnapshot.toObject()` will see
  the biggest improvement.
```

I would think that all our users use these methods, so I changed "users of" to "calls to".